Sources/BraintreeCard/Public/BraintreeCard/BTCard.h

```diff
@@ -10,6 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  A convenience initializer for creating a card tokenization request.
 */
+// Consider removing this and just using the properties. Clarifies how to make cvv only nonces?
```

Comment:
I like the idea of removing these initializers in favor of using the properties.

Good callout - we've seen a handful of questions on how to make a cvv-only nonce. I think it is worth adding a snippet to the docs on clarifying how to make a cvv-only nonce.

Reply:
Update in #604